test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -236,24 +236,24 @@ public void TestJsonSchemaConformance(string description, string schema, string
         [InlineData("false", "[]",
             "{'message_severity':'warning','code':'boolean-schema-failed','message':'Boolean schema validation failed for ''.','line':1,'column':1}")]
 
-        // dependencies validation
-        [InlineData("{'dependencies': {}}", "{}", "")]
-        [InlineData("{'dependencies': {'key1': ['key2']}}", "{'key1' : 1, 'key2' : 2}", "")]
-        [InlineData("{'dependencies': {'key1': ['key2']}}", "{}", "")]
-        [InlineData("{'dependencies': {'key1': ['key2']}}", "{'key1' : 1}",
+        // dependentSchemas validation
```

Comment:
Please also keep some tests using `dependencies` since they should also be supported

Reply:
Sure, add some dependencies back.